TombEngine/Specific/configuration.h

```diff
@@ -8,7 +8,7 @@ using namespace TEN::Math;
 
 // Directories
 
-constexpr auto REGKEY_ROOT	   = "Software\\TombEngine\\1.1.0";
+constexpr auto REGKEY_ROOT	   = "Software\\TombEngine\\1.7.0";
```

Comment:
Probably better idea is to fetch version number using `GetProductOrFileVersion()`.